open-banking-accelerator/components/com.wso2.openbanking.accelerator.common/src/main/java/com/wso2/openbanking/accelerator/common/config/OpenBankingConfigParser.java

```diff
@@ -581,6 +582,8 @@ private void buildAllowedSubscriptions() {
                                 .map(String::trim)
                                 .collect(Collectors.toList());
                         allowedAPIs.put(scopeName, rolesList);
+                    } else if (StringUtils.isEmpty(rolesStr))  {
+                        allowedAPIs.put(scopeName, Collections.emptyList());
```

Comment:
```suggestion
                    } else {
                        allowedAPIs.put(scopeName, Collections.emptyList());
```
Since the if condition checks rolesStr is not empty there's no need to check whether it's empty again in the else clause

Reply:
Resolved in [88d0ac2](https://github.com/wso2/financial-open-banking/pull/62/commits/88d0ac2834e0ceba81ceeca4486dd9ac52acd3cc)